src/help.rs

```diff
@@ -105,6 +108,7 @@ impl Help {
                     .style(Style::default().fg(Color::White))
             })
             .collect();
+        let rows_len = rows.len();
```

Comment:
this should work
```
let rows_len = self.keys.len().saturating_sub(self.block_height - 6);
```

Reply:
That worked!

c3afd1c0021e669036ae87e3cfb8ad1150a8b79b